index.html

```diff
@@ -79,6 +83,173 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div>
+              <ul class="cardTags">
+                <li class="tagText">Ruby on rails</li>
+                <li class="tagText">CSS</li>
+                <li class="tagText">JavaScript</li>
+                <li class="tagText">HTML</li>
+              </ul>
+            </div>
+            <button class="sectionBtn">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="innerSection" id="aboutSection">
+        <h3 class="sectionTitle">About Me</h3>
+        <p class="aboutMeText">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="sectionBtn">Get My Resume</button>
+        <div class="sectionGrid">
+          <div class="sectionCard aboutCard">
+            <div class="iconHolder">
+              <img src="asset/image//icons/langauge.png" alt="langauge icons" />
+            </div>
+            <h3 class="sectionCardTitle">Languages</h3>
+            <div>
+              <ul class="cardTags">
+                <li class="tagText aboutTags">JavaScript</li>
+                <li class="tagText aboutTags">Ruby</li>
+                <li class="tagText aboutTags">HTML</li>
+                <li class="tagText aboutTags">CSS</li>
+              </ul>
+            </div>
+          </div>
+          <div class="sectionCard aboutCard">
+            <div class="iconHolder">
+              <img
+                src="asset/image//icons/framework.png"
+                alt="Frameworks card icons"
+              />
+            </div>
+            <h3 class="sectionCardTitle">Frameworks</h3>
+            <div>
+              <ul class="cardTags">
+                <li class="tagText aboutTags">Bootstrap</li>
+                <li class="tagText aboutTags">Ruby on rails</li>
+                <li class="tagText aboutTags">RSpec</li>
+                <li class="tagText aboutTags">Capybara</li>
+                <li class="tagText aboutTags">Selenium</li>
+              </ul>
+            </div>
+          </div>
+          <div class="sectionCard aboutCard">
+            <div class="iconHolder">
+              <img src="asset/image//icons/langauge.png" alt="langauge icons" />
+            </div>
+            <h3 class="sectionCardTitle">Skills</h3>
+            <div>
+              <ul class="cardTags">
+                <li class="tagText aboutTags">Codekit</li>
+                <li class="tagText aboutTags">GitHub</li>
+                <li class="tagText aboutTags">Codepen</li>
+                <li class="tagText aboutTags">Gitlab</li>
+                <li class="tagText aboutTags">Terminal</li>
+              </ul>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer, the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul` instead `div`, each of them in a `li`. Check out the code below:
```html
 <ul class="sectionGrid">
          <li class="sectionCard aboutCard">
            <div class="iconHolder">
              <img src="asset/image//icons/langauge.png" alt="langauge icons" />
            </div>
            <h3 class="sectionCardTitle">Languages</h3>
            ....
          </li>
  </ul>
```

Reply:
Thank you.